apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -117,7 +122,13 @@ const SectionsTable = React.createClass({
         </thead>
         <tbody>
           {sections.map((section, index) =>
-            <tr style={styles.row} key={index}>
+            <tr
+              style={{
+                ...(!index%2 && styles.lightRow),
+                ...(index%2 && styles.darkRow), ...styles.row
```

Comment:
nit: styles.row should be on new line

Realize I said to do it the way you have it, but I also wonder if the following would be marginally easier to understand:
```
...(index % 2 === 0 ? styles.lightRow : styles.darkRow)
...styles.row
```
thoughts?